CMakeLists.txt

```diff
@@ -16,6 +16,12 @@ project (TensorComprehensions C CXX)
 
 # Display compiler flags for build modes: Release/Debug
 message(STATUS "BUILD_TYPE: ${CMAKE_BUILD_TYPE}")
+message(STATUS "CMAKE_CXX_COMPILER_VERSION is ${CMAKE_CXX_COMPILER_VERSION}")
+if(NOT CMAKE_CXX_COMPILER_VERSION VERSION_GREATER "5.4")
```

Comment:
@ttheodor , what do you think of this? I didn't put specific version thinking that 4.9 for example could also work. for example 4.8.4 and 4.8.5 are both okay.

Reply:
LG, but I'd move `-Wall` outside the conditional.